src/sdk/pynni/nni/feature_engineering/gradient_selector/requirements.txt

```diff
@@ -0,0 +1,5 @@
+numpy==1.14.3
+scikit-learn==0.20.0
+scipy==1.1.0
+torch==1.1.0
+pytest
```

Comment:
better to use `nnictl package install` to install dependencies

Reply:
Does any compressor install their dependency by `nnictl package install`?  I could take them as example.

Reply:
they do not require more dependencies, if users want to compress a pytorch model, we assume they have installed pytorch. It is better to use `nnictl package install` for convenience, but you can keep it for now.

Reply:
I want to keep it.